### ISSUE TITLE
Buggy EnderIO with Gregtech Drums

### ISSUE THREAD

**Issue by SkyrixR** [MindBrain27/Sky-Greg#42]
Hey, I was buffering my Lava with Aluminium Drums and for some weird reason enderio fluid pipes (both ender and normal) don´t insert on aluminium drums but it does on Steel and Bronze drums, already fixed it but just saying ! :D

Tried from all sides except the bottom side as the Quest Guide says :D
![Captura de pantalla 2024-01-21 043221](https://github.com/MindBrain27/Sky-Greg/assets/157075330/58ac676b-845e-449d-8aec-af560e998053)

**Comment by MindBrain27** [comment]
I was losing my mind yesterday with an Aluminium Drum and Ender IO Conduits lol, very odd that only Aluminium has an issue I didn't notice that part. I'll probably make a quest for the Aluminium Drum for now stating it can be a little buggy. I imagine that's in the list of GT fixes.

EDIT - You seem to be able to pump out of Aluminium Drums but not into them with Conduits.

**Comment by SkyrixR** [comment]
Yeah, u can pump out but not into them with EnderIO conduits, but meh, Im buffering 40 lava furnaces like crazy
